extensions/bundle/README.md

```diff
@@ -0,0 +1,50 @@
+# Phylum Bundler extension
+
+A [Phylum CLI] extension that checks your [bundle] dependencies through
+[Phylum] before installing them.
+
+## Installation
+
+This is a pre-installed extension and may be available without any additional
+action. If, for some reason, this extension is not already available, follow
+these steps to install it:
+
+Clone the repository and install the extension via the Phylum CLI.
+
+```console
+git clone https://github.com/phylum-dev/cli
+phylum extension install cli/extensions/bundle
+```
+
+## Basic usage
+
+Prepend `phylum` to your `bundle` command invocations:
+
+```console
+phylum bundle install my-package  # This will be checked by Phylum!
+```
+
+Or set up an alias in your shell to make it transparent:
+
+```console
+alias bundle="phylum bundle"
+bundle install my-package  # This will be checked by Phylum!
+```
+
+## How it works
+
+When invoking `phylum bundle`, subcommands that would modify the `package.json`,
+`npm-shrinkwrap.json`, or `package-lock.json` files will trigger a Phylum
+analysis.
+
+- If the analysis is successful, the corresponding changes will be applied.
+- If the analysis is unsuccessful because some of the new dependencies don't
+  meet the required project thresholds, the command will fail.
```

Comment:
We should update all of these... "Project thresholds" haven't existed for a while now

Reply:
Should probably open a separate issue/PR for this.

Reply:
Created https://github.com/phylum-dev/cli/issues/1186 for this.